scripts/generate-wiki-markdown.js

```diff
@@ -0,0 +1,248 @@
+  for (const distName of distNames) {
+    const seenPkgs = []
+    const repoPkgs = Object.keys(dependencies.dist[distName]).filter(name => name.startsWith(`${pkgNamespace}/`) && pkgNames.includes(name))
+
+    while (repoPkgs.length) {
+      const repoPkg = repoPkgs.shift()
+
+      seenPkgs.push(repoPkg)
+      repoPkgs.push(...Object.keys(dependencies.pkgs[repoPkg]).filter(name => name.startsWith(`${pkgNamespace}/`) && pkgNames.includes(name) && !seenPkgs.includes(name)))
+    }
+
+    for (const repoPkg of seenPkgs) {
+      Object.keys(dependencies.pkgs[repoPkg]).forEach(name => (dependencies.pkgs[repoPkg][name] = repoPkg))
+
+      dependencies.dist[distName] = {
+        ...dependencies.pkgs[repoPkg],
+        ...dependencies.dist[distName]
+      }
+    }
+
+    allDependencies.push(...Object.keys(dependencies.dist[distName]).filter(name => !pkgNames.includes(name)))
+  }
+
+  // create a table with each unique dependency as row and its version per dist
+  const table = [['Dependency', ...distNames]]
+  allDependencies.sort()
+  allDependencies
+    // remove duplicate entries
+    // this assumes that within the monorepo all packages use the same version
+    // this script is not meant for validating your deps => use renovate or similar
+    .filter((depName, index, arr) => index === arr.findIndex(name => name === depName))
+    .forEach((depName) => {
+      const npmKey = `${depName}-npm`
+      vars[npmKey] = `https://www.npmjs.com/package/${depName}`
+
+      const createShield = (distName) => {
+        const shieldKey = `${depName}-${distName}-shield`
+        vars[shieldKey] = `https://img.shields.io/npm/dependency-version/${dependencies.dist[distName][depName]}/${depName}?style=flat-square`
+        return `![${shieldKey}][${shieldKey}]`
+      }
+
+      table.push([`[${depName}][${npmKey}]`, ...distNames.map(distName => dependencies.dist[distName][depName] ? createShield(distName) : '')])
+    })
+
+  return table
+}
+
+function createBadgeTable (type, entries, vars) {
+  const table = [['Package', ...Object.values(branches).map(b => b.title)]]
+
+  for (const name of entries) {
+    const fullName = type === 'distributions' ? name : `${pkgNamespace}/${name}`
+    const linkKey = `${fullName}-href`
+    vars[linkKey] = `https://www.npmjs.com/package/${fullName}`
+
+    const row = [`[${fullName}][${linkKey}]`]
+
+    for (const branch of branches) {
+      let badge
+      if (generateOption === 'dependencies') {
+        badge = createDependencyBadge({ branch, name, fullName, type, vars })
+      } else {
+        badge = createPackageSizeBadge({ branch, name, fullName, type, vars })
+      }
+
+      row.push(badge)
+    }
+    table.push(row)
+  }
+
+  return table
+}
+
+async function main () {
+  const distPath = path.join(__dirname, '..', 'distributions')
+  const pkgsPath = path.join(__dirname, '..', 'packages')
+
+  const distributions = fs.readdirSync(distPath)
+  const packages = fs.readdirSync(pkgsPath)
```

Comment:
Maybe we can use [getWorkspacePackages](https://github.com/nuxt/nuxt.js/blob/dev/scripts/package.js#L169) helper?

Reply:
Same as above :)